<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleEarthEngineDockWidgetBase</class>
 <widget class="QDockWidget" name="GoogleEarthEngineDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>704</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>437</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>312</width>
    <height>400</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>Google Earh Engine</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_home">
       <attribute name="title">
        <string>Home Plugin</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWebView" name="home_plugin_webview">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_catalog">
       <attribute name="title">
        <string>GEE Catalog</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="previous_catalog">
             <property name="text">
              <string>previous</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="forward_catalog">
             <property name="text">
              <string>next</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="search_in_catalog">
             <property name="placeholderText">
              <string>Search</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWebView" name="catalog_webview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="zoomFactor">
           <double>1.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_doc_api">
       <attribute name="title">
        <string>GEE Doc API</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="previous_doc_api">
             <property name="text">
              <string>previous</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="forward_doc_api">
             <property name="text">
              <string>next</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="search_in_doc_api">
             <property name="placeholderText">
              <string>Search</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWebView" name="doc_api_webview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="zoomFactor">
           <double>1.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>previous_catalog</sender>
   <signal>clicked()</signal>
   <receiver>catalog_webview</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forward_catalog</sender>
   <signal>clicked()</signal>
   <receiver>catalog_webview</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forward_doc_api</sender>
   <signal>clicked()</signal>
   <receiver>doc_api_webview</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previous_doc_api</sender>
   <signal>clicked()</signal>
   <receiver>doc_api_webview</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
